OpenRA.Mods.Common/Traits/BotModules/BotModuleLogic/BaseBuilderQueueManager.cs

```diff
@@ -151,6 +151,15 @@ bool TickQueue(IBot bot, ProductionQueue queue)
 						location = possibleBuilding.Actor.Location + possibleBuilding.Trait.Info.Offset;
 					}
 				}
+				else if (actorInfo.HasTraitInfo<LineBuildInfo>())
+				{
+					orderString = "LineBuild";
+
+					if (baseBuilder.Info.WallTypes.Contains(actorInfo.Name))
+						type = BuildingType.Wall;
```

Comment:
`type` doesn't seem to be used afterwards, so we can remove this here.

Reply:
Removed it.